index.ts

```diff
@@ -49,19 +49,32 @@ app.post("/auth/login", async (req, res) => {
 })
 
 app.get("/auth/callback", async (req, res) => {
-  const { code, error_description } = req.query;
+  const { code, error_description, idp_initiated_login } = req.query;
   if (error_description) {
     return res.status(400).json({ message: error_description });
   }
 
-  const { user } = await scalekit.authenticateWithCode({
-    code: code as string,
-    redirectUri: process.env.AUTH_REDIRECT_URI!,
-  });
+  if (idp_initiated_login) {
+    const {connection_id, organization_id, login_hint, relay_state} = await scalekit.getIdpInitiatedLoginClaims(idp_initiated_login as string);
+    const url = scalekit.getAuthorizationUrl(
+      process.env.AUTH_REDIRECT_URI!,
+      {
+        connectionId: connection_id,
+        organizationId: organization_id,
```

Comment:
@srinivaskarre-sk - Does one need both the connectionId and organizationId parameters to construct Authorization URL?